.github/actions/deploy-happy-stack/action.yml

```diff
@@ -22,6 +26,10 @@ inputs:
   env:
     description: "The environment to update or create"
     required: false
+  flags:
```

Comment:
For now I would rather not add support for additional flags, unless other people think otherwise we should drop these

Reply:
Sounds good!